TASagentTwitchBot.Core/Config/BotConfiguration.cs

```diff
@@ -79,18 +80,18 @@ public class AuthConfiguration
 
         public AuthDegree TryCredentials(string password, out string authString)
         {
-
-            if (password == Admin.Password)
+            Cryptography.ComparePassword(password, Admin.Password);
```

Comment:
A redundant compare? I'm guessing this was typed out when adapting the new methods, but then left in. :)